### DISCUSSION TITLE
Is there anyway to implement the grid system in the grapsjs instead of canvas or within canvas

### DISCUSSION THREAD

**Discussion by divsbhalala** [GrapesJS/grapesjs#4807] [Q&A]
I would like to add the grid system to use with the grapsjs I added the example of that  https://react-grid-layout.github.io/react-grid-layout/examples/15-drag-from-outside.html

please let me know if is there any way for this type of implementation

**Comment by artf** [comment]
Take a look at this plugin: https://github.com/Ju99ernaut/grapesjs-plugin-toolbox

**Reply by divsbhalala** [reply]
I would like to implement react-grid-layout that provide the resize and drag and drop feature I need same feature
